src/software/physics/euclidean_to_wheel.h

```diff
@@ -0,0 +1,131 @@
+#pragma once
+
+#include <Eigen/Dense>
+
+/**
+ * Vector representation of 2D Euclidean space.
+ *
+ * 0: X
+ * 1: Y
+ * 2: W/Rotation
+ */
+typedef Eigen::Vector3d EuclideanSpace_t;
+
+/**
+ * Vector representation of the robot wheel space.
+ *
+ * 0: Front Right
+ * 1: Front Left
+ * 2: Back Left
+ * 3: Back Right
+ */
+typedef Eigen::Vector4d WheelSpace_t;
+
+class EuclideanToWheel
+{
+   public:
+    EuclideanToWheel() = default;
```

Comment:
nit: we only want to construct this class with the control_loop_frequency_Hz
```suggestion
    EuclideanToWheel() = delete;
```